cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -144,7 +145,7 @@ func Run(c *config.CompletedConfig) error {
 		}
 	}
 
-	run := func(stop <-chan struct{}) {
+	run := func(runCtx context.Context) {
```

Comment:
same nit as above. s/runCtx/ctx/

Reply:
It clashes with `ctx, err := CreateControllerContext(c, rootClientBuilder, clientBuilder, runCtx.Done())` below.

Reply:
Renamed.